builtin/logical/postgresql/backend_test.go

```diff
@@ -228,7 +250,7 @@ func testAccStepReadCreds(t *testing.T, b logical.Backend, name string) logicalt
 			}
 
 			userRows := returnedRows()
-			if userRows != 2 {
+			if userRows < 2 {
```

Comment:
With multiple permissions we cant rely on there being specifically two, but there should always be two or more